### PR TITLE
Fix arm_tilt for L1

### PR DESCRIPTION
**Pull request by CharmaineWONG2**

### CODE REVIEW

**Comment by kazewong** [comment]
@CharmaineWONG2 Would you mind adding the result of a quick test to show this fixed the inaccuracies in detector response?

**Comment by CharmaineWONG2** [comment]
The following is the test for accuracy of `jimgw.single_event.detector.GroundBased2G.antenna_pattern`. It compares the antenna response of `jim` and that of `lal`. `atol = 1e-7` is the best accuracy we can get. Since the the detector settings from `jim` follow that from `bilby` instead of `lal`, this accuracy should be acceptable.
```
from jimgw.single_event.detector import H1, L1, V1
import lal
import lalsimulation
import jax.numpy as jnp
import numpy as np

trial = 1000
ifo_names = ['H1', 'L1', 'V1']
ifos = [H1, L1, V1]
lal_prefixes = {'H1': 'H1', 'L1': 'L1', 'V1': 'V1'}
jim_outputs = jnp.zeros((trial, 6))
lal_outputs = jnp.zeros((trial, 6))

for n, ifo_name in enumerate(ifo_names):
    response = lalsimulation.DetectorPrefixToLALDetector(lal_prefixes[ifo_name]).response
    ifo = ifos[n] 
    for _ in range(trial):
        ra = 2. * jnp.pi * np.random.uniform()
        dec = jnp.pi * np.random.uniform() - np.pi / 2.
        psi = jnp.pi * np.random.uniform()
        gmst = jnp.pi * np.random.uniform()
        lal_outputs = lal.ComputeDetAMResponse(response, ra, dec, psi, gmst)
        jim_outputs = ifo.antenna_pattern(ra, dec, psi, gmst)                     
        
        assert (jnp.allclose(lal_outputs[0], jim_outputs["p"], atol = 1e-7) & jnp.allclose(lal_outputs[1], jim_outputs["c"], atol = 1e-7))
    print(f"{ifo_name} passed")
```
